doc/src/docs/InterfaceDeveloper/InterfaceDeveloper.md

```diff
@@ -1108,7 +1108,7 @@ Licensing
 
 In order to make efficient distribution of your interface, you should consider licensing EnergyPlus.  First, you must license/register to use EnergyPlus.  Since we are distributing EnergyPlus via the World Wide Web, at no charge, it will be easy to get your hands on a copy for testing out your interface.  However, you may wish to understand more of the internals and, to make a complete distribution package, will need to at least execute a distribution license.
 
-http://www.eere.energy.gov/buildings/energyplus/energyplus_licensing.html contains the details on licensing EnergyPlus for commercial or non-commercial use.
+http://apps1.eere.energy.gov/buildings/energyplus/energyplus_licensing.cfm contains the details on licensing EnergyPlus for commercial or non-commercial use.
```

Comment:
@rglazer2 Things are moving.  Please use this URL instead
https://energyplus.net/licensing